platform-http-proxy/src/main/java/org/codice/ddf/platform/http/proxy/HttpProxy.java

```diff
@@ -81,25 +90,29 @@ public void startProxy() throws Exception {
                 host = "localhost";
             }
             endpointName = ((HttpProxyServiceImpl) httpProxyService)
-                    .start("0.0.0.0:" + httpPort, "https://" + host + ":" + httpsPort, 1000, true);
+                    .start("0.0.0.0:" + httpPort, "https://" + host + ":" + httpsPort, 120000, true,
```

Comment:
@stustison, @AzGoalie I think this is causing the problem. Looks like the unit tests are expecting 1000 like eq(1000), but the implementation here has 120000. So if that is still the appropriate number than I believe we need to update the unit tests.